doc/api/cli.markdown

```diff
@@ -0,0 +1,163 @@
+# Command Line Options
+
+<!--type=misc-->
+
+Node.js comes with a wide variety of CLI options. These options expose built-in
+debugging, multiple ways to execute scripts, and other helpful runtime options.
+
+To view this documentation as a manual page in your terminal, run `man node`.
+
+
+## Synopsis
+
+`node [options] [v8 options] [script.js | -e "script"] [arguments]`
+
+`node debug [script.js | -e "script" | <host>:<port>] …`
+
+`node --v8-options`
+
+Execute without arguments to start the [REPL][].
```

Comment:
This assumes knowledge in operating a terminal which is mostly fine - maybe we can make this more accessible by adding a line saying:

```bash
$ node
>
```

Or something like that - I think it's reasonable to assume there are many windows Node users who are not really used to the command line very much.